L1Trigger/Phase2L1Jets/plugins/L1SeedConeJetProducer.cc

```diff
@@ -0,0 +1,100 @@
+#include "L1Trigger/Phase2L1Jets/interface/L1SeedConeJetProducer.hh"
+
+PFJet makeJet(std::vector<l1t::PFCandidate> parts){
+
+    l1t::PFCandidate seed = parts.at(0);
+
+    auto sumpt = [](float a, const l1t::PFCandidate& b){
+        return a + b.pt();
+    };
+
+    // Sum the pt
+    float pt = std::accumulate(parts.begin(), parts.end(), 0., sumpt);
+
+    // pt weighted d eta
+    std::vector<float> pt_deta;
+    pt_deta.resize(parts.size());
+    std::transform(parts.begin(), parts.end(), pt_deta.begin(), [&seed](const l1t::PFCandidate &part){
+            return part.pt() * (part.eta() - seed.eta());});
+    // Accumulate the pt weighted etas. Init to the seed eta, start accumulating at begin()+1 to skip seed
+    float eta = std::accumulate(pt_deta.begin()+1, pt_deta.end(), seed.eta());
+
+    // pt weighted d phi
+    std::vector<float> pt_dphi;
+    pt_dphi.resize(parts.size());
+    std::transform(parts.begin(), parts.end(), pt_dphi.begin(), [&seed](const l1t::PFCandidate &part){
+            return part.pt() * (part.phi() - seed.phi());});
+    // Accumulate the pt weighted phis. Init to the seed phi, start accumulating at begin()+1 to skip seed
+    float phi = std::accumulate(pt_dphi.begin()+1, pt_dphi.end(), seed.phi());
+
+    PFJet jet(pt, eta, phi);
```

Comment:
for eta and phi, shouldn't there be a division by pt somewhere?
 (e.g. in the transform there could be `part.pt()/pt`)